### ISSUE TITLE
Manage mailing list

### ISSUE THREAD

**Issue by rajido** [BioSchemas/specifications#27]
Think about how to create and manage mailing list. Maybe one per team and another general one for all?

**Comment by martin-nc** [comment]
Here are some options:
1. Use the comments in GitHub.
2. Set up a Google Group and use that for communications.
3. Use a trendy collaboration tool like Slack: https://slack.com/. People rave about this one.
4. Set up a free forum (phpBB etc) on a bit of spare web space. Probably not worth the bother.
5. Use plain emails like we do at the moment.

I wouldn't mind trying the trendy option, but that's just me - trendy all over.

**Comment by rajido** [comment]
@martin-nc @frenkxs and I did talk about this. We think the more basic entry level would be a mailing list and we would like to make it easy for anybody willing to join. Not everybody will be happy to have an account in GitHub or Slack, everybody has an email though. We proposed to create a mailing list per team using the bioschemas.org domain name. So the mailing list would be as follow:
- community@bioschemas.org 
- event@bioschemas.org
- trainingmaterial@bioschemas.org
- organization@bioschemas.org
- standard@bioschemas.org
- person@bioschemas.org

We proposed to manage the mailing list using Google apps using the group functionality. The team leads will have access to the administration of the mailing lists.

We also proposed to make each mailing list visible in their corresponding team page #30.
